src/operations.jl

```diff
@@ -17,6 +17,14 @@ function -(op :: AbstractLinearOperator{T}) where T
   LinearOperator{T}(op.nrow, op.ncol, op.symmetric, op.hermitian, prod, tprod, ctprod)
 end
 
+# y ⇽ α * op * x + β * y
+function mul!(y :: AbstractVector, op :: AbstractLinearOperator, x :: AbstractVector, α :: Number, β :: Number)
```

Comment:
`beta = 0` is not defined properly. It should `fill(y, 0)`, or something like that.